samples/sqltransport-nhpersistence/Core_8/Receiver/OrderLifecycleSagaData.cs

```diff
@@ -2,7 +2,9 @@
 using NServiceBus;
 
 public class OrderLifecycleSagaData :
+#pragma warning disable NSB0012 // Saga data classes should inherit ContainSagaData
```

Comment:
Is this a preferred approach? Why not change it inherit the `ContainSagaData` class instead?

Reply:
I wasn't sure if that was possible with NHibernate. I checked and the members are virtual. I will update the sample.